tools/plugins/initial-rbac/src/main/kotlin/net/corda/cli/plugin/initialRbac/commands/VNodeCreatorSubcommand.kt

```diff
@@ -27,7 +28,9 @@ class VNodeCreatorSubcommand : HttpRpcCommand(), Callable<Int> {
         // vNode related
         "Create vNode" to "POST:/api/v1/virtualnode",
         "Get all vNodes" to "GET:/api/v1/virtualnode",
-        "Update vNode" to "PUT:/api/v1/virtualnode/$VNODE_SHORT_HASH_REGEX" // TBC
+        "Get a vNode" to "GET:/api/v1/virtualnode/$VNODE_SHORT_HASH_REGEX",
+        "Update vNode" to "PUT:/api/v1/virtualnode/$VNODE_SHORT_HASH_REGEX", // TBC
+        "Update virtual node state" to "PUT:/api/v1/virtualnode/$VNODE_SHORT_HASH_REGEX/state/${RbacKeys.VNODE_STATE_REGEX}"
```

Comment:
Minor:
```suggestion
        "Update virtual node state" to "PUT:/api/v1/virtualnode/$VNODE_SHORT_HASH_REGEX/state/${VNODE_STATE_REGEX}"
```
to be consistent with the rest. `import` also needs to be tweaked.

Reply:
Nice, thanks. That's done now.